lib/squake/util.rb

```diff
@@ -14,11 +14,11 @@ def self.encode_parameters(params)
       params.map do |k, v|
         case v
         when Array
-          v.map { |e| "#{url_encode(k.to_s)}[]=#{url_encode(e.to_s)}" }.join('&')
+          v.map { "#{url_encode(k.to_s)}[]=#{url_encode(_1.to_s)}" }
```

Comment:
we can use `it` instead of `_1` when upgrade to ruby `3.4.0`:
https://www.ruby-lang.org/en/news/2024/12/25/ruby-3-4-0-released/

Reply:
That would force the users to upgrade to at least Ruby 3.4.x as well though if they use this gem. That's a breaking change.

Reply:
For the internal codebase, valid comment, though :-)